internal/hubspot/hubspotutil/hubspotutil.go

```diff
@@ -20,6 +20,9 @@ var SurveyFormID = "a86bbac5-576d-4ca0-86c1-0c60837c3eab"
 // TrialFormID is ID for the request trial form.
 var TrialFormID = "0bbc9f90-3741-4c7a-b5f5-6c81f130ea9d"
 
+// HappinessFeedbackFormID is the ID for a Happiness survey
```

Comment:
I think most comments end with a `.` in our codebase

```suggestion
// HappinessFeedbackFormID is the ID for a Happiness survey.
```